tests/org.eclipse.epsilon.ecore.delegates.test.acceptance/src/org/eclipse/epsilon/ecore/delegates/test/acceptance/Employee_Operation_AllReprotsTest.java

```diff
@@ -0,0 +1,54 @@
+package org.eclipse.epsilon.ecore.delegates.test.acceptance;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.epsilon.ecore.delegates.test.employee.Employee;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class Employee_Operation_AllReprotsTest {

```

Comment:
Class names should be in camel case for consistency e.g. EmployeeOperationAllReportsTest (also notice small typo in Repotrs).